modular_skyrat/modules/modular_items/lewd_items/code/lewd_helpers/human.dm

```diff
@@ -210,106 +187,80 @@
 /mob/living/carbon/human/proc/has_ears(required_state = REQUIRE_GENITAL_ANY)
-	var/obj/item/organ/peepee = getorganslot(ORGAN_SLOT_EARS)
-	if(peepee)
-		switch(required_state)
-			if(REQUIRE_GENITAL_ANY)
-				return TRUE
-			if(REQUIRE_GENITAL_EXPOSED)
-				if(get_item_by_slot(ITEM_SLOT_EARS))
-					return FALSE
-				else
-					return TRUE
-			if(REQUIRE_GENITAL_UNEXPOSED)
-				if(!get_item_by_slot(ITEM_SLOT_EARS))
-					return FALSE
-				else
-					return TRUE
-			else
-				return TRUE
-	return FALSE
+	var/obj/item/organ/genital = getorganslot(ORGAN_SLOT_EARS)
+	if(!genital)
+		return FALSE
+	switch(required_state)
+		if(REQUIRE_GENITAL_ANY)
+			return TRUE
+		if(REQUIRE_GENITAL_EXPOSED)
+			return !get_item_by_slot(ITEM_SLOT_EARS)
+		if(REQUIRE_GENITAL_UNEXPOSED)
+			return get_item_by_slot(ITEM_SLOT_EARS)
+		else
+			return TRUE
 
+/// Returns true if the human has accessible ear sockets for the parameter. Accepts any of the `REQUIRE_GENITAL_` defines.
 /mob/living/carbon/human/proc/has_earsockets(required_state = REQUIRE_GENITAL_ANY)
-	var/obj/item/organ/peepee = getorganslot(ORGAN_SLOT_EARS)
-	if(!peepee)
-		switch(required_state)
-			if(REQUIRE_GENITAL_ANY)
-				return TRUE
-			if(REQUIRE_GENITAL_EXPOSED)
-				if(get_item_by_slot(ITEM_SLOT_EARS))
-					return FALSE
-				else
-					return TRUE
-			if(REQUIRE_GENITAL_UNEXPOSED)
-				if(!get_item_by_slot(ITEM_SLOT_EARS))
-					return FALSE
-				else
-					return TRUE
-			else
-				return TRUE
-	return FALSE
+	var/obj/item/organ/genital = getorganslot(ORGAN_SLOT_EARS)
+	if(!genital)
+		return FALSE
+	switch(required_state)
+		if(REQUIRE_GENITAL_ANY)
+			return TRUE
+		if(REQUIRE_GENITAL_EXPOSED)
+			return !get_item_by_slot(ITEM_SLOT_EARS)
+		if(REQUIRE_GENITAL_UNEXPOSED)
+			return get_item_by_slot(ITEM_SLOT_EARS)
+		else
+			return TRUE
 
+/// Returns true if the human has accessible eyes for the parameter. Accepts any of the `REQUIRE_GENITAL_` defines.
 /mob/living/carbon/human/proc/has_eyes(required_state = REQUIRE_GENITAL_ANY)
-	var/obj/item/organ/peepee = getorganslot(ORGAN_SLOT_EYES)
-	if(peepee)
-		switch(required_state)
-			if(REQUIRE_GENITAL_ANY)
-				return TRUE
-			if(REQUIRE_GENITAL_EXPOSED)
-				if(get_item_by_slot(ITEM_SLOT_EYES))
-					return FALSE
-				else
-					return TRUE
-			if(REQUIRE_GENITAL_UNEXPOSED)
-				if(!get_item_by_slot(ITEM_SLOT_EYES))
-					return FALSE
-				else
-					return TRUE
-			else
-				return TRUE
-	return FALSE
+	var/obj/item/organ/genital = getorganslot(ORGAN_SLOT_EYES)
+	if(!genital)
+		return FALSE
+	switch(required_state)
+		if(REQUIRE_GENITAL_ANY)
+			return TRUE
+		if(REQUIRE_GENITAL_EXPOSED)
+			return !get_item_by_slot(ITEM_SLOT_EYES)
+		if(REQUIRE_GENITAL_UNEXPOSED)
+			return get_item_by_slot(ITEM_SLOT_EYES)
+		else
+			return TRUE
 
+/// Returns true if the human has accessible eye sockets for the parameter. Accepts any of the `REQUIRE_GENITAL_` defines.
 /mob/living/carbon/human/proc/has_eyesockets(required_state = REQUIRE_GENITAL_ANY)
```

Comment:
This is literally the same as `has_eyes`, unless I'm blind.

Reply:
Wait yea, why is that even a thing?

Also lmao 
![image](https://user-images.githubusercontent.com/106692773/192360834-3ef7e751-875f-443b-a367-eda3b2eeb075.png)